mater/lib/src/stores/filestore.rs

```diff
@@ -137,20 +238,33 @@ where
         Config::Balanced {
             chunk_size,
             tree_width,
-        } => balanced_import(source, output, chunk_size, tree_width).await,
+            raw,
+        } => {
+            if raw {
+                let maybe_cid = balanced_import(source, output, chunk_size, tree_width).await?;
+                let cid = maybe_cid.ok_or(Error::EmptyRootsError)?;
+                Ok(cid)
+            } else {
+                let cid = balanced_import_unixfs(source, output, chunk_size, tree_width).await?;
+                Ok(cid)
+            }
```

Comment:
The signatures match already, no need for those additions
```suggestion
            if raw {
                balanced_import(source, output, chunk_size, tree_width).await
            } else {
                balanced_import_unixfs(source, output, chunk_size, tree_width).await
            }
```